mesheryctl/pkg/utils/helpers.go

```diff
@@ -137,6 +138,7 @@ const (
 	cmdFilterView               cmdType = "filter view"
 	cmdPattern                  cmdType = "pattern"
 	cmdPatternView              cmdType = "pattern view"
+	cmdPatternExport            cmdType = "pattern export"
 	cmdApp                      cmdType = "app"
```

Comment:
@Jougan-0 I wonder if we can't delete all reference to "app" given it's prior deprecation.

Reply:
I can look into it. Opening a issue now.

Reply:
Done.